Library/Homebrew/cli/parser.rb

```diff
@@ -123,7 +121,11 @@ def initialize(&block)
 
         @args = Homebrew::CLI::Args.new
 
-        @command_name = caller_locations(2, 1).first.label.chomp("_args").tr("_", "-")
+        @command_name = if ENV["HOMEBREW_SORBET_RUNTIME"]
+          caller_locations(5, 1) # Sorbet runtime checking adds 3 methods to the stack
+        else
+          caller_locations(2, 1)
+        end.first.label.chomp("_args").tr("_", "-")
```

Comment:
I'm fine with this. An alternative might be something like:

```suggestion
        @command_name = caller_locations.select { |location| location.path.exclude?("/gems/sorbet-runtime-") }
                                        .second.label.chomp("_args").tr("_", "-")
```

which is less sensitive to future sorbet-runtime changes.

Reply:
I agree. That seems like the better solution from a maintainability perspective.